packages/types/src/product/service.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  FilterableProductCollectionProps,
+  FilterableProductProps,
+  FilterableProductTagProps,
+  FilterableProductVariantProps,
+  ProductCollectionDTO,
+  ProductDTO,
+  ProductTagDTO,
+  ProductVariantDTO,
+} from "./common"
+import { FindConfig } from "../common"
+import { SharedContext } from "../shared-context"
+
+export interface IProductService<
```

Comment:
TODO: rename to `IProductModuleService`?

Reply:
done